api/payload.go

```diff
@@ -168,3 +168,16 @@ type Signal struct {
 type ErrorResponse struct {
 	Message string `json:"msg"`
 }
+
+// LogEntry is the payload for the StreamLog data.
+type LogEntry struct {
+	// Source is the source of the log entry. One of "shim" or "runtime".
+	Source string `json:"source"`
```

Comment:
Great. It might be a good idea to add a snippet showing the new log format to one of the commits? I'd still like a pid in there to make debugging easier but as long as it is possible to distinguish between clients, I'm fine with this for the time being.